test/ClangImporter/macro-functions.swift

```diff
@@ -0,0 +1,11 @@
+// Other ClangImporter tests verify that function-like C macros are imported
+// into Swift so that identifiers are resolved correclty. This test verifies
+// these can be used in executables, and don't crash at runtime.
+
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: %target-swiftc_driver %s -import-objc-header %S/Inputs/macro-functions.h -o %t/macro-functions
```

Comment:
Please use `%target-swift-build` and `%target-run` for this, so that it works properly on iOS, and please add `REQUIRES: execution_test` so that it isn't run for cross-compilation cases. The test/Interpreter/ directory is probably a more appropriate place to put the test, too; we tend to cluster execution tests there.